Content/default/.config/dotnet-tools.json

```
@@ -3,15 +3,15 @@
   "isRoot": true,
   "tools": {
     "paket": {
-      "version": "6.0.0-alpha014",
+      "version": "6.0.0-beta8",
       "commands": [
         "paket"
       ]
     },
-    "fake-cli": {
-      "version": "5.19.1",
+    "fable": {
+      "version": "3.0.0",
```

Would it be a problem to go straight to the current version?
```suggestion
      "version": "3.1.2",
```